### PR TITLE
Migrating old Argo docs and applying minor updates

### LINKED ISSUE #48: [DOC] Tutorial on using Argo Workflows
[Current documentation](https://docs.qhub.dev/en/latest/source/admin_guide/argo-workflows.html)

Need to review the current documentation for accuracy and transform into a tutorial aimed at end users.

### LINKED ISSUE #49: [DOC] How to enable and configure Argo-Workflow
[Current documentation](https://docs.qhub.dev/en/latest/source/admin_guide/argo-workflows.html)

Need to review the current documentation for accuracy and transform into a how to doc aimed at teaching admins how to enable it on nebari

### PR DESCRIPTION
**Pull request by kcpevey**
This PR migrates over the old Argo docs and also provides some updates that are coming from PR https://github.com/nebari-dev/nebari/pull/1639 .

## Reference Issues or PRs

fixes #48

fixes #49

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [x] Documentation Update

## Testing

- [x] Did you test the pull request locally?

#### Text styling

- [ ] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [ ] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [ ] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [ ] This content adheres to the Nebari style guides.

#### Non-text content

- [x] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

File: docs/docs/tutorials/using-argo.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+id: using-argo
+title: Automating workflows with Argo
```
Comment (pavithraes) [inline docs/docs/tutorials/using-argo.md:3]:
```suggestion
title: Automate workflows with Argo
```

File: docs/docs/tutorials/using-argo.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+description: Argo workflow management
+---
+
+# Argo Workflows
```
Comment (pavithraes) [inline docs/docs/tutorials/using-argo.md:7]:
```suggestion
# Automate workflows with Argo
```

File: docs/docs/tutorials/using-argo.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+Argo Workflows is an open source container-native workflow engine for orchestrating parallel jobs on Kubernetes. Argo
+workflows comes enabled by default with Nebari deployments.
+
+## Accessing Argo Server
```
Comment (pavithraes) [inline docs/docs/tutorials/using-argo.md:12]:
```suggestion
## Access Argo Server
```

File: docs/docs/tutorials/using-argo.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+If Argo Workflows is enabled, users can access argo workflows server at: `your-nebari-domain.com/argo`. Log in via
+Keycloak with your usual credentials.
+
+## Submitting a Workflow via Argo Server
```
Comment (pavithraes) [inline docs/docs/tutorials/using-argo.md:17]:
```suggestion
## Submit a workflow via Argo Server
```

File: docs/docs/tutorials/using-argo.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+You can submit a workflow by clicking "SUBMIT NEW WORKFLOW" on the landing page assuming you have the appropriate
+permissions.
+
+![Argo Server Landing Page](/img/tutorials/argo_server_landing_page.png)
+
+## Submitting a workflow via Argo CLI
```
Comment (pavithraes) [inline docs/docs/tutorials/using-argo.md:24]:
```suggestion
## Submit a workflow via Argo CLI
```

File: docs/docs/how-tos/setup-argo.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
+---
+id: setup-argo
+title: Setting up Argo Workflows
```
Comment (pavithraes) [inline docs/docs/how-tos/setup-argo.md:3]:
```suggestion
title: Set up Argo Workflows
```

File: docs/docs/how-tos/setup-argo.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+description: Argo Workflow Setup
+---
+
+# Argo Workflows
```
Comment (pavithraes) [inline docs/docs/how-tos/setup-argo.md:7]:
```suggestion
# Set up Argo workflows
```

File: docs/docs/how-tos/setup-argo.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+Argo Workflows is an open source container-native workflow engine for orchestrating parallel jobs on Kubernetes. Argo
+workflows comes enabled by default with Nebari deployments.
+
+## Accessing Argo Server
```
Comment (pavithraes) [inline docs/docs/how-tos/setup-argo.md:12]:
```suggestion
## Access Argo Server
```

File: docs/docs/how-tos/setup-argo.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+argo_workflows:
+  enabled: true
+  overrides:
+    controller:
+      extraEnv:
+        - name: foo
+          value: bar
+```
+
+## Disabling Argo Workflows
```
Comment (pavithraes) [inline docs/docs/how-tos/setup-argo.md:34]:
```suggestion
## Disable Argo Workflows
```

**Review by pavithraes — CHANGES_REQUESTED** [review]
@kcpevey Thank you for working on this, it looks good to me. 

I've mainly just added style-guide-related suggestions for consistency. Other than that, I think the `using-argo.md` can be moved to the how-tos section -- mainly because I think tutorials are intended to be to complete end-to-end examples? What do you think?

I do agree that we can merge this migrating PR and make updates as we go. :)

**Comment by kcpevey** [comment]
@pavithraes your requested changes have been made. This is ready for final review and merge!

**Pull request merged**